samgeo/text_sam.py

```diff
@@ -342,6 +342,9 @@ def predict(
 
         if boxes.nelement() == 0:  # No "object" instances found
             print("No objects found in the image.")
+            mask_overlay = np.zeros_like(
+                image_np[..., 0], dtype=dtype
+            )  # Create an empty mask overlay
             return
```

Comment:
I guess the `return` statement needs to be removed here. Otherwise, the `mask_overlay` will not be returned

Reply:
oops, my bad. Corrected.